### ISSUE TITLE
[BUG/ISSUE] Possible incorrect units of mixing ratio in ObsPack output

### ISSUE THREAD

**Issue by chfite** [geoschem/geos-chem#333]
I have noticed in my ObsPack output that the mixing ratio variable "q" appears to be in units of g/kg based on the magnitudes I am seeing (values ranging 5-17). The ObsPack output gave this a unit label of kg/kg. So, either the data needs to be converted to kg/kg, or the netCDF unit label for q in ObsPack should list g/kg instead. I am not sure if this is only an issue depending on the meteorology one is using, but I am using GEOSFP and GC 12.6.0.

**Comment by yantosca** [comment]
I took a look at this.  You are right, the obspack_q variable (specific humidity) should be g/kg, because the SPHU field in GEOS-Chem is g/kg, not kg/kg.   We will fix this for 12.9.0.  

In the code below, we need to change:
```
    ! Specific humidity
    CALL NcDef_Variable( fId, 'q', NF_FLOAT, 1, dims_1d, vId                )
    attVal = 'mass_fraction_of_water_inair'
    CALL NcDef_Var_Attributes( fId, vId, 'long_name', TRIM(attVal)          )
    attVal = 'kg water (kg air)^-1'
    CALL NcDef_Var_Attributes( fId, vId, 'units',     TRIM(attVal)          )
```
to
```
    ! Specific humidity
    CALL NcDef_Variable( fId, 'q', NF_FLOAT, 1, dims_1d, vId                )
    attVal = 'mass_fraction_of_water_inair'
    CALL NcDef_Var_Attributes( fId, vId, 'long_name', TRIM(attVal)          )
    attVal = 'g water (kg air)^-1'
    CALL NcDef_Var_Attributes( fId, vId, 'units',     TRIM(attVal)          )
```

**Comment by msulprizio** [comment]
This is now fixed in ed226e9ce729f1a55a586083661eef052390fdf4 and will be included in 12.9.0.
